pyanaconda/storage/installerstorage.py

```diff
@@ -0,0 +1,2161 @@
+#
```

Comment:
I think that the original `osinstall` is a better name for the module then the `installerstorage`.